float32 x
float32 y 
float32 vel
float32 hdg
float32 vel_l
float32 vel_r